fedbiomed/common/environ.py

```diff
@@ -183,3 +168,10 @@ def set_environment(self):
             "MPSPDZ_CERTIFICATE_PEM",
             os.path.join(self._values["CONFIG_DIR"], public_key)
         )
+
+        # make `secagg_insecure_validation` optional in config file (and avoid changing config file version)
+        # by using a `fallback` value
+        secagg_insecure_validation = self.config.get('security', 'secagg_insecure_validation', fallback='true')
```

Comment:
Please find the suggestion for reducing the size of the line and a shorter comment.  

```suggestion
        # Optional secagg_insecure_validation optional in config file
        secagg_insecure_validation = self.config.get(
            'security', 'secagg_insecure_validation', fallback='true')
```